cypress/utils/utils.ts

```diff
@@ -577,7 +577,7 @@ export function uploadFile(fileName: string): void {
 export function navigate_to_application_inventory(tab?): void {
     cy.get("h1", { timeout: 5 * SEC }).then(($header) => {
         if (!$header.text().includes("Application inventory")) {
-            selectUserPerspective("Developer");
+            selectUserPerspective("Migration");
```

Comment:
Hi @kpunwatk . All files should use const "migration" 
and NOT "Migration" or "Administration" 

So that if this values change in future const is the only class we will have to change

Reply:
Thanks @sshveta for pointing this out. Done !